tests/test_web_sendfile_functional.py

```diff
@@ -334,9 +343,9 @@ async def handler(request):
 
     resp = await client.get("/")
     assert resp.status == 200
-    assert resp.headers.get("Content-Encoding") == expect_encoding
-    assert resp.headers["Content-Type"] == "text/plain"
-    assert await resp.read() == HELLO_AIOHTTP
+    assert resp.headers.get("Content-Encoding") is None
```

Comment:
I think this is more explicit as:
```suggestion
    assert "Content-Encoding" not in resp.headers
```

Reply:
Actually, I considered that, but with that change, the log will only tell you it's true if the assertion fails. This way it tells you what the header is actually set to. It's done this way in another test above.